packages/cli/src/daemon/sse-feed.ts

```diff
@@ -0,0 +1,42 @@
+import type { Observable } from 'rxjs'
+import type { Response } from 'express'
+import type { DiagnosticsLogger } from '@ceramicnetwork/common'
+
+export class SseFeed<TInput> {
```

Comment:
Can we add a class comment describing what this does?  To someone who isn't familiar with Server Sent Events they might not even know what "Sse" means.

Reply:
I don't know why github isn't showing it here, but if you go to the file you'll see a comment